api/filters/plans_filter_by_visibility.go

```diff
@@ -50,7 +50,7 @@ func (vf *PlanFilterByVisibility) FilterMatchers() []web.FilterMatcher {
 	return []web.FilterMatcher{
 		{
 			Matchers: []web.Matcher{
-				web.Path(web.ServicePlansURL + "/**"),
+				web.Path(web.ServicePlansURL + "/*"),
```

Comment:
Why is the second star removed? This filter will not be attached for getting single plan. Same goes for others.

Reply:
Well according to docs it should be attached - a single star means up to the next separator /. Its removed because we dont want these filters on the operations subapi

Reply:
yes, I missed the part that we don't need them on the operations subapi